packages/api/es/indexer.js

```diff
@@ -401,30 +453,18 @@ function logHandler(event, context, cb) {
  * Lambda function handler for sns2elasticsearch
  *
  * @param  {Object} event - incoming message sns
- * @param  {Object} context - aws lambda context object
- * @param  {function} cb - aws lambda callback function
  * @returns {Promise} undefined
  */
-function handler(event, context, cb) {
+async function handler(event) {
```

Comment:
Refactored this function because the way that callbacks were being used here was eating exceptions without any logging output.